helmfile.d/prometheus.yaml

```diff
@@ -7,7 +7,7 @@ releases:
     chart: prometheus-community/prometheus
     wait: true
     timeout: 300
-    version: 14.4.1
+    version: 14.5.0
```

Comment:
No changelog for the charts but the version code diff (<https://github.com/prometheus-community/helm-charts/compare/prometheus-14.4.1...prometheus-14.5.0>) shows the following changes for this chart: https://github.com/prometheus-community/helm-charts/compare/prometheus-14.4.1...prometheus-14.5.0#diff-a7d2f872af425efb23e365d063a23bdfcc33de8728c2b7bb4d536b7b8e80981a
=> It's an enhancement with the status `Completed` supported for the kubernetes scrapping for slow pods, which introduced the support for "Kubernetes Jobs" pods.